scss/datetime/_theme.scss

```diff
@@ -75,6 +81,71 @@
     // Timepicker
     .k-timepicker {}
 
+    // Timepicker header
+    .k-time-header {
+        padding: $time-header-padding;
+
+        .k-title {
+            font-weight: bold;
+        }
+
+        .k-time-now {
+            color: $accent;
+            cursor: pointer;
+
+            &:hover,
+            &:focus {
+                color: $link-hover-text;
+            }
+        }
+    }
+
+    // Timepicker content
+    .k-time-list-wrapper {
+        background-color: $popup-bg;
+
+        .k-title {
+            @include appearance( header );
```

Comment:
`appearance(header)` sets color, border-color and **background color**. Yet, the latter is overriden few lines below. Also, consider using `background-color` instead of `background`.

If you only need to set the color, do so.